apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Quansight Labs awarded three CZI EOSS Cycle 5 Grants"
+author: jaime-rodriguez-guerra
```

Comment:
Shouldn't all recipient be listed?

Reply:
It's not possible at the moment. There's an open tracking issue to add multiple authors once the blog system is updated to resolve this issue. I've added this post there.